thrift-datanode/src/main/thrift/datanode.thrift

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+include "rpc.thrift"
+namespace java org.apache.iotdb.service.rpc.thrift
+namespace py iotdb.thrift.rpc
+
+typedef i32 int
+typedef i64 long
+
+service ManagementIService {
+    /**
+      * Config node will create a schema region on a list of data nodes.
+      *
+      * @param data nodes of the schema region, and schema region id generated by config node
+    **/
+    rpc.TSStatus createSchemaRegion(1:list<int> DataNodeID, 2:int schemaRegionID)
+
+    /**
+      * Config node will create a data region on a list of data nodes.
+      *
+      * @param data nodes of the data region, and data region id generated by config node
+    **/
+    rpc.TSStatus createDataRegion(1:list<int> DataNodeID, 2:int dataRegionID)
```

Comment:
We could package this parameters into a thrift struct. So that it would be easier for interface update.